src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {ButtonPanel} from './ButtonPanel';
+import {Display} from './Display';
+import {calculate} from '../logic/calculate';
```

Comment:
The main objective of this milestone is to create a `calculate.js` and `operate.js` files in the logic folder. 
Please check this files to git.